modules/ROOT/pages/resource-management.adoc

```diff
@@ -30,6 +30,12 @@ contentType=layout
 
 If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.
 
+[NOTE]
+
+====
+To insert line breaks in a page.properties file, end the line that preceeds the line break with a "\".
+====
```

Comment:
I suggest to provide an example of properties source file in the NOTE block. The user will be able to better understand what we mean.

Reply:
ok. @mariellebonita ; can you help by providing an example following the case you have solved on this very topic? That would be great.

Reply:
Thanks Marielle!